spec/turn_message_spec.rb

```diff
@@ -0,0 +1,126 @@
+require "turn_message"
+require "mastermind"
+require "turn"
+
+RSpec.fdescribe TurnMessage do
```

Comment:
Try to not commit tests with focus flags such as `fdescribe` or `fit`. If merged, such code will not run all other tests, which is a big loss.

Reply:
Okay, I will be more careful next time